### PR TITLE
Only read the master on clone

### PR DESCRIPTION
**Pull request by blittle**
This is because the release flow of the sample app is merging the
default branch (integration) into master. There may be lots of things
merged into the default branch that we don't yet want to release.

### CODE REVIEW

File: test/generator.test.ts [hunk 1]
```diff
@@ -10,11 +10,17 @@ describe('CreateGenerator', () => {
 
   before(async () => {
     const tmpGitRepo = tmp.dirSync().name;
-
-    fs.writeFileSync(`${tmpGitRepo}/package.json`, '{"name: "test"}');
+    console.log('hi', `${tmpGitRepo}/package.json`);
```
Comment (jbegleiter-sf) [inline test/generator.test.ts]: Should this be here?

Comment (blittle) [reply test/generator.test.ts]: What be where? 😜

File: test/generator.test.ts [hunk 2]
```diff
@@ -10,11 +10,22 @@ describe('CreateGenerator', () => {
...
+    fs.writeFileSync(`${tmpGitRepo}/package.json`, '{"name": "test"}');
 
     await git(tmpGitRepo).init()
-      .catch(console.log);
+      .catch(console.error.bind(console));
+
+    await git(tmpGitRepo).addConfig('user.name', 'Some one')
+      .catch(console.error.bind(console));
+
+    await git(tmpGitRepo).addConfig('user.email', 'some@one.com')
```
Comment (jbegleiter-sf) [inline test/generator.test.ts]: Should we add a docs update RE git global config..?

Comment (blittle) [reply test/generator.test.ts]: No, this is only necessary for the tests. Me forcing a branch on checkout, git doesn't say there is a branch until there is a commit. So for the tests, I just add a single commit, but to commit there needs to be a user present in the git repo.

**Pull request merged**